packages/graph/hash_graph/lib/interval-ops/src/interval.rs

```diff
@@ -98,8 +98,10 @@ pub trait Interval<T>: Sized {
         let rhs_lower = other.lower_bound();
         let rhs_upper = other.upper_bound();
 
-        // Range A:    [-----] | [-----]
-        // Range B: [-----]    |    [-----]
+        // Example |      A     |     B
+        // ========|============|============
+        // Range A |    [-----] | [-----]
+        // Range B | [-----]    |    [-----]
```

Comment:
Does this comment need an output/result row?

Reply:
IMO it doesn't because it just shows how ranges could overlap